Examples/PythonExamples/apic_example_01.py

```
@@ -1,4 +1,4 @@
-from pyCubbyFlow import * # lgtm [py/polluting-import]
+from pyCubbyFlow import *
```

Ruff (0.15.20)

[error] `from pyCubbyFlow import *` used; unable to detect undefined names

(F403)